Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -120,12 +136,17 @@ public class Client: CustomStringConvertible {
                                   httpClient: chttpClient,
                                   jwks: jwks,
                                   tokenHandler: tokenHandler)
+        let closure = { authCode, authState, completion in
```

Comment:
Maybe give the name some context? `makeTokenCallback`?

Reply:
Will fix that!